xbmc/cores/RetroPlayer/rendering/VideoRenderers/RPRendererOpenGL.cpp

```diff
@@ -120,119 +59,3 @@ CRPRendererOpenGL::CRPRendererOpenGL(const CRenderSettings &renderSettings, CRen
   // Initialize CRPRendererOpenGLES
   m_clearColour = m_context.UseLimitedColor() ? (16.0f / 0xff) : 0.0f;
```

Comment:
This line is from before GLES supported limited color. Can it be moved to GLES now?

Reply:
GLES still doesn't support limited color

Reply:
k